sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
It is needed for the testbots to pull in external dependencies.  http://drupal.stackexchange.com/questions/181793/adding-a-contrib-dependency-to-simpletest